models/br_sfb_sicar/code/architecture.py

```
@@ -0,0 +1,346 @@
+
+# ---- reusable column builders ---------------------------------------------
+
+DIR_UF = "br_bd_diretorios_brasil.uf:sigla"
+DIR_MUNI = "br_bd_diretorios_brasil.municipio:id_municipio"
+DIR_DATA = "br_bd_diretorios_data_tempo.data:data"
+
+
+def c(
+    name,
+    src,
+    type,
```

Ruff (0.16.1)

[error] Function argument `type` is shadowing a Python builtin

(A002)